forecast/issue.md

```diff
@@ -0,0 +1,120 @@
+---
+description: How to issue a forecast as a buyer
+---
+
+# Issue a new forecast
+
+As buyer you can send a new forecast to your supplier.
+
+{% hint style="info" %}
+A forecast cannot be updated, a forecast will always be appended.
+{% endhint %}
+
+{% hint style="warning" %}
```

Comment:
Probably best to review this page when it's merged... Telling from the .md file I don't see anything weird...

Reply:
It has previews again, see https://app.gitbook.com/s/-LNyJ9UuwLCjTvA2sqQR-887967055/~/revisions/3gKr9fQuJRqwQ1CnaSTD/

Reply:
@roy-tc Last check? https://app.gitbook.com/s/-LNyJ9UuwLCjTvA2sqQR-887967055/~/revisions/eiWXVO38sOT1iLGCCPEI/